sysapps/device_capabilities/cpu_info_provider_win.cc

```diff
@@ -4,14 +4,27 @@
 
 #include "xwalk/sysapps/device_capabilities/cpu_info_provider.h"
 
+#include <pdh.h>
+
 #include "base/logging.h"
 
 namespace xwalk {
 namespace sysapps {
 
+static PDH_HQUERY cpuQuery;
+static PDH_HCOUNTER cpuTotal;
+
+void CPUInfoProvider::init() {
+  PdhOpenQuery(NULL, NULL, &cpuQuery);
```

Comment:
only some can be replaced, the second NULL here is a DWORDPTR and thus doesn't compile with nullptr. Not sure I want to add a cast here.